src/EnergyPlus/WindowComplexManager.cc

```diff
@@ -1538,14 +1538,22 @@ namespace WindowComplexManager {
             Geom.SolSkyWt(I) = SkyWeight(Geom.sInc(J));
         }
         WtSum = sum(Geom.SolSkyWt({1, NSky}));
-        Geom.SolSkyWt({1, NSky}) /= WtSum;
+        if (WtSum > DataGlobalConstants::rTinyValue) {
```

Comment:
This is a one-time setup process for each complex window, so some extra checks are not a computational issue.